src/contracts/abis/index.js

```diff
@@ -29,6 +30,7 @@ export {
   RewardSplitterFactoryAbi,
   VestingEscrowFactoryAbi,
   MintTokenControllerAbi,
+  OtherTokenVaultAbi,
```

Comment:
Do we need this abi? It is completely similar to the VaultAbi so we can use it instead of the new one